src/ErrorDisplay/ErrorDisplay.js

```diff
@@ -1,14 +1,24 @@
 import './ErrorDisplay.css';
 import icon from '../Assets/browser.png';
-import Header from '../Header/Header';
+import { Link } from 'react-router-dom';
 
 const ErrorDisplay = ({ errorMessage }) => {
 
+  let displayText = '';
+  let button = null;
+
+  if (errorMessage) {
```

Comment:
I like this if else statement here. It will be beneficial to a user to know what is going wrong.